composer.json

```
@@ -23,7 +23,7 @@
         "knplabs/knp-snappy-bundle": "^1.7",
         "myclabs/php-enum": "^1.7",
         "sylius/resource-bundle": "^1.6",
-        "sylius/sylius": "~1.11.4",
+        "sylius/sylius": "~1.11.4 || ~1.12.0",
```

```suggestion
        "sylius/sylius": "^1.11.4",
```
but I'm not sure if your version isn't more explicit 🤔